src/hibernate-resume/hibernate-resume.c

```diff
@@ -1,58 +1,55 @@
 /* SPDX-License-Identifier: LGPL-2.1-or-later */
 
 #include <errno.h>
-#include <stdio.h>
 #include <sys/stat.h>
 
-#include "alloc-util.h"
 #include "devnum-util.h"
-#include "fileio.h"
 #include "initrd-util.h"
 #include "log.h"
+#include "main-func.h"
+#include "parse-util.h"
+#include "sleep-util.h"
 
-int main(int argc, char *argv[]) {
+static const char *arg_resume_device = NULL;
+static uint64_t arg_resume_offset = 0;
```

Comment:
similar here, add a comment `/* in memory pages */`

the thing is simply that throughout our codebase we prefer to unify sizes/time values and so on onto a common base, i.e. bytes, µs and so on. This one here is an exception. (an OK one, given that apparently everything around the hibernation logic operates in pages, and this is directly visible because of the kernel cmdline option). But exceptions really deserve comments.